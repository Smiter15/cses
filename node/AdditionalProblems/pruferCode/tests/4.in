10
7 10 5 5 5 5 1 7
