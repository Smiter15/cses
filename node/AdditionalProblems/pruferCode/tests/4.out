7 2
10 3
5 4
5 6
5 8
5 9
1 5
7 1
7 10 
